src/Registration/Common/Perform2DTranslationRegistrationWithMeanSquares/Code.py

```diff
@@ -27,95 +27,70 @@
           "<outputImagefile> <differenceImageAfter> <differenceImageBefore>")
     sys.exit(1)
 
-fixedImage = sys.argv[1]
-movingImage = sys.argv[2]
-outputImage = sys.argv[3]
-differenceImageAfter = sys.argv[4]
-differenceImageBefore = sys.argv[5]
+fixedImageFile = sys.argv[1]
+movingImageFile = sys.argv[2]
+outputImageFile = sys.argv[3]
+differenceImageAfterFile = sys.argv[4]
+differenceImageBeforeFile = sys.argv[5]
+
 
 Dimension = 2
-PixelType = itk.F
+PixelType = itk.ctype('float')
 
 FixedImageType = itk.Image[PixelType, Dimension]
 MovingImageType = itk.Image[PixelType, Dimension]
 
-TransformType = itk.TranslationTransform[itk.D, Dimension]
-OptimizerType = itk.RegularStepGradientDescentOptimizerv4[itk.D]
-
-MetricType = itk.MeanSquaresImageToImageMetricv4[
-    FixedImageType, MovingImageType]
-
-RegistrationType = itk.ImageRegistrationMethodv4[
-    FixedImageType, MovingImageType]
-
-metric = MetricType.New()
-optimizer = OptimizerType.New()
-transform = TransformType.New()
-registration = RegistrationType.New()
-
-registration.SetMetric(metric)
-registration.SetOptimizer(optimizer)
-registration.SetInitialTransform(transform)
+fixedImage = itk.imread(fixedImageFile, PixelType)
+movingImage = itk.imread(movingImageFile, PixelType)
 
-FixedLinearInterpolatorType = itk.LinearInterpolateImageFunction[
-    FixedImageType, itk.D]
-MovingLinearInterpolatorType = itk.LinearInterpolateImageFunction[
-    MovingImageType, itk.D]
 
-fixedInterpolator = FixedLinearInterpolatorType.New()
-movingInterpolator = MovingLinearInterpolatorType.New()
-
-metric.SetFixedInterpolator(fixedInterpolator)
-metric.SetMovingInterpolator(movingInterpolator)
+TransformType = itk.TranslationTransform[itk.D, Dimension]
+initialTransform = TransformType.New()
 
-FixedImageReaderType = itk.ImageFileReader[FixedImageType]
-MovingImageReaderType = itk.ImageFileReader[MovingImageType]
-fixedImageReader = FixedImageReaderType.New()
-movingImageReader = MovingImageReaderType.New()
+optimizer = itk.RegularStepGradientDescentOptimizerv4[itk.D].New(
+        LearningRate=4,
```

Comment:
I assume that the CapWords convention is coming from the C++ and there is no way to change it ;)
I think that PEP8 would prefer something like `learning_rate`.

Reply:
We can do it! :-)

Please review this patch:

  http://review.source.kitware.com/#/c/23213/